lib/matplotlib/markers.py

```diff
@@ -201,7 +201,7 @@ class MarkerStyle:
     # TODO: Is this ever used as a non-constant?
     _point_size_reduction = 0.5
 
-    def __init__(self, marker=None, fillstyle=None):
+    def __init__(self, marker=None, fillstyle=None, normalize_path=True):
```

Comment:
make this keyword-only?

Reply:
Sure, will do. I think the name is up to discussion anyways.